main.go

```diff
@@ -75,12 +52,13 @@ type CredentialsCommand struct {
 type CreateCommand struct {
 	*ClusterCommand
 
-	Wait bool
-
 	// Options passed along to Carina's API
 	Nodes     int
 	AutoScale bool
 
+	// Whether to wait until the cluster is created (or errored)
+	Wait bool
```

Comment:
Moved this down because of aligncheck stating we could fit this structure in less bytes. I would think the compiler handles this, but I did it anyway.

```
/shrug
```